vpr/src/route/annotate_routing.cpp

```diff
@@ -3,12 +3,14 @@
  * from VPR to OpenFPGA
```

Comment:
Expand comment to explain (briefly) what annotating routing results to OpenFPGA means.